.cirrus.yml

```diff
@@ -70,18 +94,19 @@ qa_task_filter_template: &QA_TASK_FILTER
     - build
   <<: *ONLY_IF_SONARSOURCE_QA
 
+# ----------------------------------------------
+# -------------------TASKS----------------------
+# ----------------------------------------------
+
 build_task:
   <<: *LINUX_3_5_CPU_7G
+  <<: *MAVEN_CACHE
   env:
-    SIGN_KEY: VAULT[development/kv/data/sign data.key]
-    PGP_PASSPHRASE: VAULT[development/kv/data/sign data.passphrase]
     # analysis on SonarCloud
-    SONAR_TOKEN: VAULT[development/kv/data/sonarcloud data.token]
     SONAR_HOST_URL: https://sonarcloud.io
+    <<: *BUILD_SECRETS
     #allow deployment of pull request artifacts to repox
     DEPLOY_PULL_REQUEST: true
-  maven_cache:
-    folder: ${CIRRUS_WORKING_DIR}/.m2/repository
   build_script:
     - source cirrus-env BUILD-PRIVATE
```

Comment:
Maybe it's not related, but I wonder why there is `BUILD-PRIVATE` if the project is public?